### PR TITLE
Create base crate, set dependencies to v0.9.43, add Tinkernet XCM configs

### PR DESCRIPTION
**Pull request by arrudagates**

### CODE REVIEW

**Comment by arrudagates** [comment]
@xlc And so it begins, I structured it similarly to `xcm-builder`, but with 2 modules exported: `kusama` and `polkadot`, where the intention is to house configs that require to exist within one of those consensus systems. For everything else that's not dependent on the relay chain, it should be declared in other modules and exported in lib.rs.

We also have to setup CI, contribution guidelines and a readme.

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+std = [
+	"log/std",
+	"parity-scale-codec/std",
+	"scale-info/std",
+	"xcm/std",
+	"xcm-executor/std",
+	"sp-std/std",
+	"sp-arithmetic/std",
+	"sp-io/std",
+  "sp-core/std",
```
Comment (xlc) [inline Cargo.toml]: indent

**Comment by xlc** [comment]
Our team can handle the CI.

Do we want this to be a multi crate setup? I will image we want the Kusama configs and Polkadot configs in two different crates so that a Kusama parachain only need to import the Kusama configs without the Polkadot configs.

Another way is feature flag. But I will want to avoid it if possible.

**Comment by arrudagates** [comment]
> Our team can handle the CI.
> 
> Do we want this to be a multi crate setup? I will image we want the Kusama configs and Polkadot configs in two different crates so that a Kusama parachain only need to import the Kusama configs without the Polkadot configs.
> 
> Another way is feature flag. But I will want to avoid it if possible.

I think keeping them on the same crate will be helpful for types that work regardless of the relay chain. Otherwise we would have 3 crates.

Maybe we don't even need feature flags, I can't imagine this would add much to compile time and type checking, and at the end of the day, everything that's unused gets tree-shaken.

**Comment by xlc** [comment]
I don't mind having 3 crates. I want to make sure it is impossible to accidentally include the wrong types from the other relaychain.

**Comment by arrudagates** [comment]
Alright, I'll convert it to a workspace of 3 crates and commit to this PR tomorrow, we can then go from there if we figure out another more efficient approach.

**Comment by arrudagates** [comment]
@xlc There we go, I have refactored it to a workspace of 3 crates: `orml-xcm-builder`, `orml-xcm-builder-kusama` and `orml-xcm-builder-polkadot`, with the first one being the general purpose non relay specific crate.

**Review by xlc — APPROVED** [review]
maybe call it shared instead of general? but I am fine either way

**Comment by arrudagates** [comment]
I agree shared would be a better name.

**Comment by arrudagates** [comment]
I'm going to merge this now!

**Pull request merged**